### DISCUSSION TITLE
javascript

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9216] [Help]
Hi everyone.
I iused this js code. but there is an error while executing

_**var s = $("Form 2.Fields.motif (e8)");
if (helper.lengthEqual(s,0)){
	return helper.stringify("NON")
}
return helper.stringify("OUI")**_

and this one 

**_var s = $("Form 2.Fields.motif (e8)");
if (helper.lengthEqual(s,0)){
	return "NON"
}
return "OUI"_**

the column type is text

![js error](https://user-images.githubusercontent.com/106697208/210552451-aad0983b-c9d7-4a78-adc4-1a91b4df1e0f.PNG)

@melohagan any help please ?

**Comment by melohagan** [comment]
Instead if doing `if (helper.lengthEqual...)` you could simply do a truthy check: `if (!s)`

**Reply by melohagan** [reply]
Also just `return "NON"` not `helper...`

**Comment by rafikApp** [comment]
Hi @melohagan.
Yes i already tried that, but it doesn't work

**Reply by melohagan** [reply]
It's hard to help without screenshots and additional context...

**Reply by rafikApp** [reply]
**_var s = $("Form.Fields.consignateur (e5)")+$("Form.Fields.nouveau consignateur (e5)")
if (!s){
	return "En attente signature"
}
else{
	return "A confirmée"
}_**

I worked with this code, but it always returns "En attente signature" whether the string is empty or not.
I think there is a problem with this line. :  **_var s = $("Form.Fields.consignateur (e5)")+$("Form.Fields.nouveau consignateur (e5)")_**
Because when I try to display the value of **_$("Form.Fields.consignateur (e5)")+$("Form.Fields.nouveau consignateur (e5)")_** with javascript using an embed nothing is displayed despite their field containing a value

**Reply by melohagan** [reply]
What data-types are `$("Form.Fields.consignateur (e5)")` and `$("Form.Fields.nouveau consignateur (e5)")` ? If you print out either one individually are you seeing the correct value?

**Reply by rafikApp** [reply]
$**("Form.Fields.consignateur (e5)")** : relationship
**$("Form.Fields.nouveau consignateur (e5)")** : text
No the same thing if I display them separately using javascript
Whereas it is displayed well with handelbars

**Reply by rafikApp** [reply]
Hi @melohagan 
I found the solution, using handelbars.
**_{{#or Form.Fields.consignateur (e5).length  Form.Fields.nouveau consignateur (e5).length}}A confirmée{{else}}En attente signature{{/or}}_**
